tutorials/rise_mnist.ipynb

```diff
@@ -18,7 +18,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    X_test = open_image(path_gray)
</pre><p>Overwriting X_test with some tensor with another shape then before (3 axes instead of 4) results in an error later on (I think).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>